frontend/packages/console-dynamic-plugin-sdk/src/extensions/console-types.ts

```diff
@@ -468,6 +468,7 @@ export type ResourceLinkProps = {
   truncate?: boolean;
   nameSuffix?: React.ReactNode;
   children?: React.ReactNode;
+  metadata?: any;
```

Comment:
`any` is not an appropriate type. We should have types defined for k8s resource metadata. (I'd need to double check where.)